### PR TITLE
Fixes #21420 - Add the name of the file in Edit file

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

**Comment by shiramax** [comment]
[test foreman]

**Comment by shiramax** [comment]
[test katello]

File: app/helpers/form_helper.rb [hunk 1]
```diff
@@ -187,7 +187,11 @@ def spinner_button_f(f, caption, action, html_options = {})
 
   def file_field_f(f, attr, options = {})
     field(f, attr, options) do
-      f.file_field attr, options
+      html = ''
+      if options[:file_name]
+        html += content_tag(:span) {"Current file os '#{options.delete(:file_name)}', upload a new file to change it" + ' ' || '' }
```
Comment (amirfefer) [inline app/helpers/form_helper.rb]: 1. you don't have to use block for span's content
2. string should be extracted for translation.
3.  typo? (os = is?)

File: app/helpers/form_helper.rb [hunk 2]
```diff
@@ -187,7 +187,11 @@ def spinner_button_f(f, caption, action, html_options = {})
 
   def file_field_f(f, attr, options = {})
     field(f, attr, options) do
-      f.file_field attr, options
+      html = ''
```
Comment (amirfefer) [inline app/helpers/form_helper.rb]: I found this unnecessary a bit.
consider  using nesting fields or inline if statement?

**Review by amirfefer — CHANGES_REQUESTED** [review]

**Review by dLobatog — APPROVED** [review]
Thanks  @shiramax  @amirfefer

**Pull request merged**